packages/app/src/AppNav.tsx

```diff
@@ -897,6 +899,9 @@ export default function AppNav({ fixed = false }: { fixed?: boolean }) {
             </Link>
           </div>
         </div>
+        <div className="d-flex justify-content-end align-items-end">
```

Comment:
nit: since this isn't super important, we can probably slap a `fs-7` in the className to make it smaller